packages/frontend-shared/src/locales/en-US.json

```diff
@@ -650,9 +650,20 @@
       "switchTestingTypeInfo": "There is {n} {testingType} test failing in this spec. To run it locally switch to {testingType} testing. | There are {n} {testingType} tests failing in this spec. To run them locally switch to {testingType} testing.",
       "switchTestingTypeAction": "Switch to {testingType} testing"
     },
-    "notLoggedIn": "Not logged in",
-    "notConnected": "Not connected",
-    "noRuns": "No runs",
+    "overPlanLimit": "Over plan limit",
+    "reachedMonthlyUsage": "You've reached the monthly usage limit for your billing plan.",
+    "upgradeYourPlan": "Upgrade your plan to continue recording runs.",
+    "upgradePlan": "Upgrade plan",
+    "contactAdmin": "Contact admin",
+    "wellDone": "Well Done!",
+    "allYourTestsPassed": "All your tests passed.",
+    "incomplete": "Incomplete",
+    "specsSkipped": "{totalSkippedSpecs} of {n} spec skipped | {totalSkippedSpecs} of {n} specs skipped",
+    "runHasNoTests": "Run has no tests",
+    "theRunStartedButNeverCompleted": "The run started, but never completed. This can happen when the run is cancelled from CI or when Cypress crashes during running tests.",
```

Comment:
/nitpick: I think the comma is unnecessary here.

Also, should it be "crashes _while_ running tests."?

Reply:
yeah I agree with both of these